### PR TITLE
[WIP] Add Recommendation File for GitHubTeacher

### LINKED ISSUE #1: GitHub Flow for @hectorsector
- [ ] Create a **branch**
- [ ] Add some *commits*
- [ ] Open a Pull Request
- [ ] Add some more commits
- [ ] Merge

### PR DESCRIPTION
**Pull request by githubteacher**
closes #1

### CODE REVIEW

**Comment by githubteacher** [comment]
🍰

File: githubteacher-bio.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## San Diego, CA
+
+### Places to Eat
+- City Tacos
```
Comment (githubteacher) [inline githubteacher-bio.md:4]: I recommend getting 3 tacos here.

File: githubteacher-bio.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+- Sushi N Joy
+- Herb n wood
```
Comment (githubteacher) [inline githubteacher-bio.md]: This is pricy but delicious

File: githubteacher-bio.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+### Places to Visit
+- Coronado Island
```
Comment (githubteacher) [inline githubteacher-bio.md:14]: What do you do here?

**Review by githubteacher — COMMENTED** [review]
THIS IS HOW I FEEL IN GENERAL

File: githubteacher-bio.md (hunk 2 above)
Comment (hectorsector) [inline githubteacher-bio.md]: Needs more 🌮

**Review by hectorsector — CHANGES_REQUESTED** [review]

File: githubteacher-bio.md (hunk 2 above)
Comment (hectorsector) [reply githubteacher-bio.md]: needs MOAR

Comment (githubteacher) [reply githubteacher-bio.md]: no

**Review by hectorsector — APPROVED** [review]

**Review by hectorsector — APPROVED** [review]
:shipit:

**Pull request merged**